02/had.py

```diff
@@ -0,0 +1,16 @@
+#----- HAD A VELBLOUD
+import sys
+
+variable = sys.argv[1]
```

Comment:
`variable` není dobrý název proměnné. Prosím změnit například na:
```suggestion
input_text = sys.argv[1]
```

Reply:
Opraveno